frontend/src/bundles/achievements/store/slice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { DataStatus } from '~/bundles/common/enums/enums.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { type AchievementResponsetDto } from '../types/types.js';
```

Comment:
typo
Also we have different naming convention for such dtos

Reply:
This is temporary til API will be ready